Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Dockerfile to build and test DeepDive inside a container
+#
+# `make build-in-container test-in-container` uses master image built by this.
+# `util/build/docker/` contains utilities relevant to this.
+FROM ubuntu
```

Comment:
This Dockerfile and DockerBuild/ scripts are for development.  For production and normal users, I think we should have a separate set of Dockerfiles that just installs the built DD binary releases with their runtime dependencies.  For that we can/should certainly fix the ubuntu version.  However, if we're running everything containerized, perhaps the host OS distro doesn't even matter.  We could just mount the DD app and run it to populate another database container.  However, yes for standalone releases, we may want to test it against various flavors of distros and versions, or just explicitly target few of them.

Justification for separate production images: The deepdive-build image is now ~4-5GiB.  However, given that the release binary is ~100MiB compressed and as the runtime dependencies are going to be much smaller than build deps, I'd say the production image can easily go under 1GiB.